test/cypress/integration/page-extension.spec.js

```diff
@@ -12,17 +14,20 @@ context('Page extension', () => {
     cy.visit(pageExtension('test-extension'))
     cy.waitForIFrame()
     cy.get(selectors.pageIFrame).captureIFrameAs('extension')
-  })
-
-  it('opens a page extension and tests navigating within the page', () => {
     cy.get('@extension')
       .find(selectors.page)
       .should('exist')
+  })
 
+  it('opens a page extension and tests navigating within the page', () => {
     cy.get('@extension')
       .find(selectors.openNewPathButton)
       .click()
 
     cy.url().should('include', 'test-extension/new')
   })
+
+  /* Reusable tests */
```

Comment:
Is this comment necessary?

Reply:
@davidfateh not necessary, but I think it's a good separator between tests specific to a location and reusable ones.